tests/ec2_standalone_tests.sh

```diff
@@ -22,4 +22,11 @@ export PYWREN_EXECUTOR=remote
 pytest -v tests/test_simple.py
 RESULT=$?
 pywren standalone terminate_instances
+
+pywren standalone launch_instances 1 --max_idle_time=10 --idle_terminate_granularity=600 --pywren_git_commit=$TRAVIS_COMMIT  --parallelism 16
+sleep 20
+export PYWREN_EXECUTOR=remote
+pytest -v tests/test_simple.py
```

Comment:
Is there a test that actually checks if 16 were run in parallel ?

Reply:
Nope, I'm not really sure how to test that.